airbyte-bootloader/src/main/java/io/airbyte/bootloader/BootloaderApp.java

```diff
@@ -72,32 +75,55 @@ public BootloaderApp(final Configs configs, final FeatureFlags featureFlags) {
    * @param configs
    * @param postLoadExecution
    */
-  public BootloaderApp(final Configs configs, final Runnable postLoadExecution, final FeatureFlags featureFlags) {
+  public BootloaderApp(final Configs configs,
+                       final Runnable postLoadExecution,
+                       final FeatureFlags featureFlags,
+                       final SecretMigrator secretMigrator) {
     this.configs = configs;
     this.postLoadExecution = postLoadExecution;
     this.featureFlags = featureFlags;
+    this.secretMigrator = secretMigrator;
   }
 
   public BootloaderApp() {
     configs = new EnvConfigs();
     featureFlags = new EnvVariableFeatureFlags();
-    postLoadExecution = () -> {
-      try {
-        final Database configDatabase =
-            new ConfigsDatabaseInstance(configs.getConfigDatabaseUser(), configs.getConfigDatabasePassword(), configs.getConfigDatabaseUrl())
-                .getAndInitialize();
-        final JsonSecretsProcessor jsonSecretsProcessor = JsonSecretsProcessor.builder()
-            .maskSecrets(!featureFlags.exposeSecretsInExport())
-            .copySecrets(true)
-            .build();
-        final ConfigPersistence configPersistence =
-            DatabaseConfigPersistence.createWithValidation(configDatabase, jsonSecretsProcessor);
-        configPersistence.loadData(YamlSeedConfigPersistence.getDefault());
-        LOGGER.info("Loaded seed data..");
-      } catch (final IOException e) {
-        e.printStackTrace();
+
+    try {
+      final Database configDatabase = new ConfigsDatabaseInstance(configs.getConfigDatabaseUser(), configs.getConfigDatabasePassword(),
+          configs.getConfigDatabaseUrl())
+              .getAndInitialize();
+      final JsonSecretsProcessor jsonSecretsProcessor = JsonSecretsProcessor.builder()
+          .maskSecrets(!featureFlags.exposeSecretsInExport())
```

Comment:
why do we set this in the bootloader? the bootloader never exports right? i guess it is fine for consistency. seems a bit unnecesarry.

Reply:
It was from a refactoring suggestion about performing a no-op secret processor. The goal was to avoid having any feature flag logic in the JsonSecretsProcessor class

Reply:
yeah. that makes sense to me. what i'm saying is that i would assume that the builder would default this to true if it weren't set. or even if we do need to explicitly set it, we could hardcode it to true, because the bootloader should never export secrets anyway.